agential/cog/prompts/react.py

```diff
@@ -232,3 +232,94 @@
 Observation 3: [States by Order of Entry into Union - Infoplease] Joining the Union | State | Entered Union | Year Settled | | North Carolina | Nov. 21, 1789 | 1660 | | Rhode Island | May 29, 1790 | 1636 | | Vermont | Mar. 4, 1791 | 1724 | 
 Thought 4: The evidence shows North Carolina entered Union in 1789, and Rhode Island entered Union in 1790, which is a year after North Carolina. 
 Action 4: Finish[Rhode Island]"""
+
+REACT_INSTRUCTION_TABMWP = """Read the following table and question to solve the problem,providing Observation that SUPPORTS the answer, or indicating if there is NOT ENOUGH INFORMATION. Action can be two types:
+(1) Python[Question], generates an executable python python code using the Sympy library using the Context. The first line of the code should be 'from sympy import *'
+(2) Finish[answer], which returns the answer and finishes the task.
+You have a maximum of {max_steps} steps.
+Here are some examples:
+{examples}
+(END OF EXAMPLES)
```

Comment:
needs to be rewritten

Reply:
I got all my prompt instructions from this repo where they build the React prompt for TABMWP. Here are all my references:

https://github.com/rakutentech/MathSensei/blob/main/src/REACT_prompts/prompts_MATH_react.py

https://github.com/rakutentech/MathSensei/blob/986c21528ad7b5719dc19f158cce0b9f2e122e60/src/custom_prompts/prompt_pg.py#L172"
-